planning.py

```diff
@@ -12,3 +11,67 @@
 import time
 import bisect
 import string
+from test.test_asyncio.test_events import noop
```

Comment:
This one is entirely my fault. Forgot to Cleanup the code before submitting the PR.
Thus these unnecessary lines.